src/components/Card/UserGroup/UserGroupCard.jsx

```diff
@@ -0,0 +1,49 @@
+import changeDayFormat from "../../../utils/changeDayFormat";
+import getDate from "../../../utils/getDate";
+import KeywordChip from "../../Chip/KeywordChip";
+import PropTypes from "prop-types";
+
+const UserGroupCard = ({ groupName, keywordList, createdAt, updatedAt }) => {
+  const createdDate = getDate(createdAt);
+  const updatedDate = getDate(updatedAt);
+
+  return (
+    <div className="flex flex-col items-start justify-center gap-3 w-full border-2 border-rose-200/80 bg-white rounded-[30px] px-20 py-10">
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">1️⃣ Group Name: </span>
+        <span className="text-rose-400 text-18">{groupName}</span>
+      </p>
+      <p className="flex items-center gap-10">
+        <span className="text-purple-300 text-20">2️⃣ Keyword List: </span>
+        <span className="text-rose-400 text-18">
+          {keywordList.map((keywordName) => {
+            <KeywordChip key={keywordName} keywordName={keywordName} />;
+          })}
```

Comment:
```suggestion
          {keywordList.map((keyword) => {
            <KeywordChip key={keyword.keywordId} keywordName={keyword.keyword} />;
          })}
```
추후에 유저 그룹 조회 API가 구현 완료되면, 해당 부분도 key를 키워드의 ID로 설정해주시면 더 좋을 것 같아요!
(저 포함 다른 팀원이 해당 TASK를 진행할 수 있으니까, 리뷰 남깁니다!)

Reply:
작성 중에 백엔드 Schema가 수정되어 친 부분인데, 의견 감사합니다. suggestion 주신대로 수정해보겠습니다~!

Reply:
네, 좋습니다 👍